plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -2025,8 +2025,7 @@ public Optional<ConnectorOutputMetadata> finishRefreshMaterializedView(
 
         String dependencies = sourceTableHandles.stream()
                 .map(handle -> (IcebergTableHandle) handle)
-                .filter(handle -> handle.getSnapshotId().isPresent())
```

Comment:
Question about this commit, if tables created by trino always have a snapshot id and CREATE MV cannot be pointed to pre-existing tables, then can this scenario arise in practise ?
Could we require that iceberg table always have a snapshot id here instead ?

Reply:
> Question about this commit, if tables created by trino always have a snapshot id and CREATE MV cannot be pointed to pre-existing tables, then can this scenario arise in practise ?

see https://github.com/trinodb/trino/commit/5c1750ea5a54a353efc9ead7ccce34c545c70d1c#r80166651

Reply:
got it, thanks
can we have a test which creates MV in trino on source iceberg table created by spark ?

Reply:
We should (as part of https://github.com/trinodb/trino/pull/4832 ...)